packages/node_modules/@ciscospark/react-container-notifications/src/container.test.js

```diff
@@ -28,7 +28,8 @@ describe('Notifications component', () => {
     username: 'username',
     message: 'message',
     avatar: 'avatar',
-    notificationId: 'abc123'
+    notificationId: 'abc123',
+    alertType: 'full'
```

Comment:
Would like to see a test to make sure 'none' alert types don't fire.

Reply:
Good call, will do